tests/support/Traits/SqlMockTrait.php

```diff
@@ -157,6 +157,42 @@ protected function mockCreate(array $selectResult, $notFillableValue): void
         );
     }
 
+    protected function mockInsertData(): void {
+        $now = Carbon::now();
```

Comment:
now function is mocking to return the static value in test, no need to call it here, you can just use this static value in tests